docs/user/access-control.md

```diff
@@ -53,17 +53,13 @@ kubectl create -n default rolebinding example-view \
 #### Write access to Applications within a namespace
 
 In order to create, update and delete Applications in a namespace, apply the
-`edit` ClusterRole in the desired namespace and the `kubeapps-repositories-read`
+`edit` ClusterRole in the desired namespace and the `$RELEASE_NAME-repositories-read`
 Role in the namespace Kubeapps is installed in. The `edit` ClusterRole should be
 available in most Kubernetes distributions, you can find more information about
 that role
 [here](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles).
 
-Additionally, we need to create a role to give read access to App Repositories and bind it to our service account.
-
 ```
-export KUBEAPPS_NAMESPACE=kubeapps
-kubectl apply -n $KUBEAPPS_NAMESPACE -f https://raw.githubusercontent.com/kubeapps/kubeapps/master/docs/user/manifests/kubeapps-repositories-read.yaml
 kubectl create -n default rolebinding example-edit \
   --clusterrole=edit \
   --serviceaccount default:example
```

Comment:
doesn't this need to be updated to point to the role using the Kubeapps release name?

Reply:
true, I missed that hidden part

Reply:
Hm - this started clean and easy... :)

My initial changes were to define the roles in the `kubeapps` namespace for *apprepositories* - `git show b7afc95fd0673a0ade26c6d59c363f291dab14d0` removed them as manifest-files, and added to the helm template in `18a156471a60473621cd97b73f66219493c0eb30` and `4cba08837be8a25459f2607faf186ff16cced707`.

However - the relevant documentation is really from line 113 and onwards (for repositories).

The section at around line 60 is about apps.

Where do we want to go with this?
Add all roles to the chart? Add cluster roles?

Reply:
You are right @davidkarlsen, this PR is getting longer than it should.

> The section at around line 60 is about apps.

Yes, but the thing is that for deploying an app the user needs to have permission to read repositories in the Kubeapps namespace so that's why these commands are duplicated here and in the section about AppRepositories (line 113).

> Where do we want to go with this?
Add all roles to the chart? Add cluster roles?

That's not needed, the only missing change is to modify the line 71 to use `s/kubeapps-repositories-read/$RELEASE_NAME-repositories-read/`. Let me change it myself to clarify it.

Reply:
This should be read now if I have not missed anything.